corehq/sql_accessors/sql_templates/get_case_ids_in_domain_1.sql

```diff
@@ -0,0 +1,31 @@
+DROP FUNCTION IF EXISTS get_case_ids_in_domain(TEXT, TEXT, TEXT[], BOOLEAN);
+
+CREATE FUNCTION get_case_ids_in_domain(
+    domain_name TEXT,
+    case_type TEXT DEFAULT NULL,
+    owner_ids TEXT[] DEFAULT NULL,
+    p_closed BOOLEAN DEFAULT NULL
+) RETURNS TABLE (case_id VARCHAR(255)) AS $$
+DECLARE
+    query_expr      TEXT := 'SELECT case_id FROM form_processor_commcarecasesql WHERE domain = $1';
+    type_filter     TEXT := ' AND type = $2';
+    owner_filter    TEXT := ' AND owner_id = ANY($3)';
+    closed_filter   TEXT := ' AND closed = $4';
+BEGIN
+    IF case_type <> '' THEN
```

Comment:
No it's not but this that `case_type is neither NULL or emtpy`: http://stackoverflow.com/a/23767625